conf/messages

```diff
@@ -74,7 +74,7 @@ pla.notAuthorised.tryAgain      = try to confirm your identity again
 pla.lockedOut.title             = Locked Out
 pla.lockedOut.pageHeading       = You have tried to confirm your identity too many times
 pla.lockedOut.paraOne           = You can try again in 24 hours.
-pla.lockedOut.paraTwo           = <p>If you have a query you can <a href="{0}" target="_blank">contact HM Revenue and Customs (opens in a new window)</a> to get help.</p>
+pla.lockedOut.paraTwo           = <p class="govuk-body">If you have a query you can <a href="{0}" target="_blank" class="govuk-link">contact HM Revenue and Customs (opens in new tab)</a> to get help.</p>
```

Comment:
The html for this `<a>` could be moved into `lockedOut.scala.html`, leaving just the content